app/views/layout.erb

```diff
@@ -1,15 +0,0 @@
-<!doctype html>
-<html>
-<head>
-  <meta charset="utf-8">
-  <title>My Application</title>
-</head>
-
-<body>
-  <!-- This is what a code comment in HTML looks like. -->
-
-  <!-- The `yield` below is where the contents of a view file are put. -->
-  <!-- Then the combined layout + view is sent back to the client. -->
-  <%= yield %>
```

Comment:
Don't delete the layout.erb file. It's helpful and allows you to DRY up your views.